CHANGELOG/CHANGELOG-1.3.md

```diff
@@ -0,0 +1,95 @@
+- Update ELK stack reference configuration to support TCP transport. ([#2387](https://github.com/antrea-io/antrea/pull/2387), [@zyiou])
+- Update Windows installation instructions. ([#2456](https://github.com/antrea-io/antrea/pull/2456), [@lzheheng])
+- Update Antrea-native policies documentation to reflect the addition of the "kubernetes.io/metadata.name" in upstream K8s. ([#2596](https://github.com/antrea-io/antrea/pull/2596), [@abhiraut])
+- Default to containerd as the container runtime in the Vagrant-based test K8s cluster. ([#2583](https://github.com/antrea-io/antrea/pull/2583), [@stanleywbwong])
+- Update AllowToCoreDNS example in Antrea-native policies documentation. ([#2605](https://github.com/antrea-io/antrea/pull/2605), [@btrieger])
+- Update actions/setup-go to v2 in all Github workflows. ([#2517](https://github.com/antrea-io/antrea/pull/2517), [@MysteryBlokHed])
+
+### Fixed
+
+- Fix panic in Agent when calculating the stats for a rule newly added to an existing NetworkPolicy. ([#2495](https://github.com/antrea-io/antrea/pull/2495), [@tnqn])
+- Fix bug in iptables rule installation for dual-stack clusters: if a rule was already present for one protocol but not the other, its installation may have been skipped. ([#2469](https://github.com/antrea-io/antrea/pull/2469), [@lzhecheng])
+- Fix deadlock in the Agent's FlowExporter, between the export goroutine and the conntrack polling goroutine. ([#2429](https://github.com/antrea-io/antrea/pull/2429), [@srikartati])
+- Upgrade OVS version to 2.14.2-antrea.1 for Windows Nodes; this version of OVS is built on top of the upstream 2.14.2 release and also includes a patch to fix TCP checksum computation when the DNAT action is used. ([#2549](https://github.com/antrea-io/antrea/pull/2549), [@lzhecheng]) [Windows]
+- Handle transient iptables-restore failures (caused by xtables lock contention) in the NodePortLocal initialization logic. ([#2555](https://github.com/antrea-io/antrea/pull/2555), [@antoninbas])
+- Query and check the list of features supported by the OVS datapath during Agent initialization: if any required feature is not supported, the Agent will log an error and crash, instead of continuing to run which makes it hard to troubleshoot such issues. ([#2571](https://github.com/antrea-io/antrea/pull/2571), [@tnqn])
+- Periodically delete stale connections in the Flow Exporter if they cannot be exported (e.g. because the collector is not available), to avoid running out-of-memory. ([#2516](https://github.com/antrea-io/antrea/pull/2516), [@srikartati])
+- Fix handling of the "reject" packets generated by the Antrea Agent in the OVS pipeline, to avoid infinite looping when traffic between two endpoints is rejected by network policies in both directions. ([#2579](https://github.com/antrea-io/antrea/pull/2579), [@GraysonWu])
```

Comment:
network policies -> Network Policies

Reply:
not changing, because this is taken from earlier CHANGELOGs and I want to keep the entry the same

Reply:
I feel we should stick to our naming convention, but you decide.